quiz.css

```diff
@@ -0,0 +1,93 @@
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+.wrapper {
+  max-width: 960px;
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+h1 {
+  margin: 10px;
+}
+
+h3, 
+h6 {
+  margin: 0;
+}
+
+h3 {
+  font-weight: normal;
+}
+
+h6{
+  font-weight: bold;
+}
+
+.card1 {
+  float: left;
+}
+
+.card2 {
+  float: left;
+  margin-left: 5%;
+}
+
+.card3 {
+  float: right;
+}
+
+.card {
+  width: 30%;
+  border: 1px solid lightblue;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.card .title {
+  border: 1px solid lightgray;
```

Comment:
**Requirement not met:** The card title has a dotted 1px border that is lightgray. Yours is solid.